### ISSUE TITLE
[REQUEST]: Subdomain For ML Nashik I mlcommunity.dev

### ISSUE THREAD

**Issue by MehulPardeshi** [ml-comm/ddns#15]
### Describe your request

We need a subdomain for our tech community as it enhances credibility, centralizes resources, and boosts engagement, making it essential for fostering a connected and professional presence.

### Subdomain

nashik 

### Purpose

The subdomain will serve as a central platform for ML Nashik, aimed at building a strong AI/ML community by providing access to resources, updates on events, and networking opportunities for members. It will support collaboration, facilitate knowledge sharing, and boost engagement within the local tech community, establishing a professional presence for ML Nashik on the web.

### Socials

GitHub: MehulPardeshi 
Linkedin: http://linkedin.com/in/mehul-pardeshi

**Comment by BRAVO68WEB** [comment]
@MehulPardeshi 

Can you share your current website?

**Comment by MehulPardeshi** [comment]
Our website has yet to be developed. Currently, our only presence on the
platform is ML Nashik on Commuddle.
Here is our LinkedIn profile:
https://www.linkedin.com/company/mlnashik/

**Comment by BRAVO68WEB** [comment]
Hey @MehulPardeshi 

We actually expect that before you apply for subdomain you must have your website pre-build.
Closing this for now, create a new issue once website is ready.
